docs/scripting/resources/weaponids.md

```diff
@@ -0,0 +1,71 @@
+---
+id: weaponids
+title: Weapon IDs
+---
+
+:::info
+
+To be used with [OnPlayerDeath](../callbacks/OnPlayerDeath), [GivePlayerWeapon](../functions/GivePlayerWeapon), [SendDeathMessage](../functions/SendDeathMessage).
+
+:::
+
+---
+| Hud Icon                                                                      | Death Reason                                          | Definition                | ID        | Slot      | Model         | Notes                                                                                                                 |
+| ----------------------------------------------------------------------------- | ----------------------------------------------------- | ------------------------- | --------- | --------- | ------------- | --------------------------------------------------------------------------------------------------------------------- |
+| ![](/images/hudIcons/fist.png)<br/>Fist                                       | ![](/images/deathIcons/death-fist.gif)                | -                         | 0         | 0         | -             | -                                                                                                                     |
+| ![](/images/hudIcons/brassKnuckles.png)<br/>Brass Knuckles                    | ![](/images/deathIcons/death-brassKnuckles.gif)       | WEAPON_BRASSKNUCKLE       | 1         | 0         | 331           | -                                                                                                                     |
+| ![](/images/hudIcons/golfClub.png)<br/>Golf Club                              | ![](/images/deathIcons/death-golfClub.gif)            | WEAPON_GOLFCLUB           | 2         | 1         | 333           | -                                                                                                                     |
+| ![](/images/hudIcons/nightStick.png)<br/>Nightstick                           | ![](/images/deathIcons/death-nightStick.gif)          | WEAPON_NITESTICK          | 3         | 1         | 334           | -                                                                                                                     |
```

Comment:
`/images/deathIcons/death-nightStick.gif` fails to load for some reason

Reply:
Not only that, these are included:
```
/images/hudIcons/sprayCan.png
/images/hudIcons/flameThrower.png
/images/hudIcons/microSMG-UZI.png
/images/deathIcons/death-microSMG-UZI.gif
```